platforms/netlify-ci-build/README.md

```diff
@@ -1,7 +1,8 @@
 # Introduction
 
-Prisma and Netlify integration
+Prisma and Netlify (beta build) integration.
 
 # Friction Points
 
 - https://github.com/netlify/build/issues/730
```

Comment:
Can we put a super short description of the Friction point here? In this case probably "`binaryTargets` needs to be `[...]`" or similar.